src/seedu/addressbook/AddressBook.java

```diff
@@ -1147,11 +1146,11 @@ private static String getUsageInfoForExitCommand() {
      * Removes sign(p/, d/, etc) from parameter string
      *
      * @param s  Parameter as a string
-     * @param sign  Parameter sign to be removed
+     * @param prefix  Parameter sign to be removed
```

Comment:
Good! Code changes should be reflected in the comments if necessary. Btw, you should probably update line 1146 to remove the mention of "sign".